[ACM-6261]: Restructuring kubevirt and AWS files to reuse 'Accessing a hosted cluster' (#5335)
open-cluster-management/rhacm-docs: clusters/hosted_control_planes/hosted_cluster_arm_aws.adoc

[#hosted-cluster-arm-aws]
= Enabling hosted control planes on an ARM64 {ocp-short} cluster

You can enable an ARM64-hosted control plane to operate with an {ocp-short} ARM64 data plane in a management cluster environment. This feature is available for hosted control planes on AWS only.

[#prerequisites-hosted-arm]
== Prerequisites

Before you begin, you must meet the following prerequisites:

* You must have an {ocp-short} cluster that was installed on a 64-bit ARM infrastructure. For more information, see link:https://console.redhat.com/openshift/install/aws/arm[Create an OpenShift Cluster: AWS (ARM)].
* You must have a HyperShift Operator that is built on a 64-bit ARM infrastructure. You can obtain a HyperShift Operator by going to the link:https://quay.io/repository/hypershift/hypershift-operator[hypershift/hypershift-operator repository] and selecting the build that has the `4.13-arm64` tag. 

> To run a hosted cluster on an ARM64 {ocp-short} cluster, take the following steps:

Reviewer: ```suggestion
To run a hosted cluster on an ARM64 {ocp-short} cluster, complete the following steps:
```

. Install the HyperShift Operator for ARM64 on the management cluster to override the default HyperShift Operator image.

+

> For example, through the hosted control planes (`hypershift`) command line interface, enter the following commands, being careful to replace the bucket name, AWS credentials, and region with your information:

Reviewer: ```suggestion
For example, through the hosted control planes (`hypershift`) command line interface, enter the following commands, and replace the bucket name, AWS credentials, and region with your information:
```

+
----
hypershift install \
--oidc-storage-provider-s3-bucket-name $BUCKET_NAME \
--oidc-storage-provider-s3-credentials $AWS_CREDS \
--oidc-storage-provider-s3-region $REGION \
--hypershift-image quay.io/hypershift/hypershift-operator:4.13-arm64
----

. Create a hosted cluster that overrides the default release image with a multi-architecture release image. 

+

> For example, through the hosted control planes (`hypershift`) command line interface, enter the following commands, being careful to replace the cluster name, node pool replicas, base domain, pull secret, AWS credentials, and region with your information:

Reviewer: similar suggestion, remove "being careful"

+
----
hypershift create cluster aws \ 
--name $CLUSTER_NAME \
--node-pool-replicas=$NODEPOOL_REPLICAS \
--base-domain $BASE_DOMAIN \
--pull-secret $PULL_SECRET \
--aws-creds $AWS_CREDS \
--region $REGION \
--release-image quay.io/openshift-release-dev/ocp-release:4.13.0-rc.0-multi
----

+
This example adds a default `NodePool` object through the `--node-pool-replicas` flag.

> . Add a 64-bit x86 `NodePool` object to the hosted cluster.

Reviewer: ```suggestion
. Add a 64-bit x_86 `NodePool` object to the hosted cluster.
```
